packaging/README-dockerhub.md

```diff
@@ -2,8 +2,6 @@
 
 The images are automatically rebuilt on new GIT tags or pushes to the master branch:
 
-`Dockerfile.fedora-nmstate-dev` by https://cloud.docker.com/u/nmstate/repository/docker/nmstate/fedora-nmstate-dev
-
 The base image contains a common base that is used both for the development
 image and for the distributed image.
 
```

Comment:
The documentation is docker hub specific for the trigger. On Quai the default settings with the repo, container spec like `/packaging/fedora-nmstate-dev` and build context `/packaging` work. Then we get tags that are named after the branch/tag that triggered the build.

Reply:
Done!